### PR TITLE
Add a validation method to mml_utils

### PR DESCRIPTION
**Pull request by asilversempirical**
This successfully discovers the wonky column in the larger MPC data set

### CODE REVIEW

File: src/mml_utils.py [hunk 1]
```diff
@@ -127,3 +129,48 @@ def to_mml(mml_json, table, generator):
...
+        # If the column is already ignored there's nothing to check
+        if typ['stattype'] is 'IGNORE':
+            continue
+        one_col_json = copy.deepcopy(mml_json)
+        one_col_json['columns'] = {col: typ}
+        # Create a temp generator
+        gen_name = uuid.uuid4().hex
+        try:
+            bdb.execute(to_mml(one_col_json, table, gen_name))
+            bdb.execute('INITIALIZE 1 MODEL FOR %s' % bql_quote_name(gen_name))
```
Comment (madeleine-empirical) [inline src/mml_utils.py]: `(bql_quote_name(gen_name),)`

**Comment by madeleine-empirical** [comment]
Test?

**Comment by asilversempirical** [comment]
Test added.

**Comment by madeleine-empirical** [comment]
LGTM, send to probcomp at your convenience.

**Comment by asilversempirical** [comment]
Opened https://github.com/probcomp/bdbcontrib/pull/154

**Pull request closed without merging**